### ISSUE TITLE
Correction to Anthology ID 2021.emnlp-main.318

### ISSUE THREAD

**Issue by Betahj** [acl-org/acl-anthology#1672]
## Revision or erratum: please add the following information**

- [√] I have attached the revised PDF or erratum to this issue
- [√] I have written a brief description of the changes below

We deleted the two Fundings in acknowledgement of this paper because they are irrelevant with our work in this paper.
[EMNLP21_2085_revision_updated_ABSA.pdf](https://github.com/acl-org/acl-anthology/files/7562404/EMNLP21_2085_revision_updated_ABSA.pdf)

**Comment by xinru1414** [comment]
@Betahj Please state the correct anthology ID for the paper.

**Comment by Betahj** [comment]
Thank you for your reminder, the Anthology ID has been modified.

**Comment by Betahj** [comment]
Duplicate of #1670

**Comment by Betahj** [comment]
Revision or erratum:
----------------------------------------------------------------------------
@xinru1414  I am so sorry to bother you again. According to the instructions of program chairs, we have updated the current camera-ready version and made several corrections. Could you please help us to update our paper with the following attachment I provided?

- [√] I have attached the revised PDF or erratum to this issue
- [√] I have written a brief description of the changes

[2021.emnlp-main.318.revised_version.pdf](https://github.com/acl-org/acl-anthology/files/7942165/2021.emnlp-main.318.revised_version.pdf)

**Comment by xinru1414** [comment]
@Betahj Please briefly described the changes. Thx!

**Comment by Betahj** [comment]
@xinru1414 I have given a brief summary of the changes we have made, please check at your convenience.

**Comment by Betahj** [comment]
I am so sorry to bother you again this time. I have given a brief summary of the changes we have made 12 days ago, but we haven't seen the revised version I provided 13 days ago on ACL Anthology. Could you please help us to check what happened and update our paper with the revised version I provided 13 days ago?
The link of our issue is:&nbsp;https://github.com/acl-org/acl-anthology/issues/1672



---Original---
From: ***@***.***&gt;
Date: Thu, Jan 27, 2022 01:59 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [acl-org/acl-anthology] Correction to Anthology ID2021.emnlp-main.318 (Issue #1672)




 
@Betahj Please briefly described the changes. Thx!
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mjpost** [comment]
Per [the instructions](https://aclanthology.org/info/corrections/), please provide a short descriptive summary of the changes. It does not need to describe every detail, just the gist. The bullet points above are too lengthy.

We process corrections roughly once a month and will do so shortly, so if this is ready in time, it will be processed with January's corrections.

**Comment by Betahj** [comment]
Thank you very much! Following the instructions, we have provided a brief description of the changes we made. Please help us update our paper with the revised version I provided 13 days ago.&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "acl-org/acl-anthology"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月8日(星期二) 晚上8:06
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [acl-org/acl-anthology] Correction to Anthology ID 2021.emnlp-main.318 (Issue #1672)





 
Per the instructions, please provide a short descriptive summary of the changes. It does not need to describe every detail, just the gist. The bullet points above are too lengthy.
 
We process corrections roughly once a month and will do so shortly, so if this is ready in time, it will be processed with January's corrections.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by xinru1414** [comment]
@Betahj it's been taken care of here: https://github.com/acl-org/acl-anthology/pull/1761 and will be alive on anthology soon
